### ISSUE TITLE
LineChart: adjustToWidth not working

### ISSUE THREAD

**Issue by markkobbert** [Abhinandan-Kushwaha/react-native-gifted-charts#387]
The adjustToWidth property doesn't seem to be working.

This code:

```
const data = [{value: 15}, {value: 30}, {value: 26}, {value: 40}];

  return (
    <View
      style={{
        backgroundColor: 'blue',
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
        borderTopWidth: 1,
      }}>
      <LineChart data={data} backgroundColor={'red'} adjustToWidth />
    </View>
  );
```

results in this:

![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/39066484/5bb5e114-df9c-4d2e-bb63-3bea62ac6321)

1. why is there space on the right side of the chart (inside the red area)
2. why is the chart not across the whole width of the parent (blue area)

**Comment by 1mike12** [comment]
I've seen the same issue. It seems to leave a bit of space on the right and doesn't take up the whole width of the parent

**Comment by Abhinandan-Kushwaha** [comment]
Hi @1mike12 @markkobbert 
Thanks for pointing this out.
It has been fixed in version `1.3.7`🎉
